bedrock/base/templates/macros.html

```diff
@@ -229,7 +229,7 @@ <h2 class="thank-you hidden">{{ _('Your download link was sent.') }}</h2>
     {% do fxa_queries.append('entrypoint_variation=' ~ entrypoint_variation) %}
   {% endif %}
 
```

Comment:
perhaps we should consider renaming the `monitor_button` macro to something more generic and passing a `product='Monitor'` or `product='Pocket'` value to change the href? It could also add a selector to the macro we could target in tests to make sure the correct href for the product is being shown.

Reply:
This will come in the next PR (I'm just keeping the scope of this PR to the original isssue for now). But honestly, I don't think we should continue to make our macros more and more complex like this, as it already feels brittle. I think something that doesn this type of product link generation makes sense, but it should be a Python helper so that we can have unit tests.

Reply:
That sounds good, I created an issue for that work here: #8143